portfolio/src/main/webapp/script.js

```diff
@@ -27,17 +27,27 @@ function addRandomSong() {
   songContainer.innerText = song;
 }
 
+
 async function showString() {
     const responseFromServer = await fetch('/hello');
     const jsonFromResponse = await responseFromServer.json();
     console.log(jsonFromResponse);
 
-    const stringContainer = document.getElementById('string-container');
-    stringContainer.innerHTML = getRandomQuote(jsonFromResponse);
+    const quoteContainer = document.getElementById('random-quote-container');
+    quoteContainer.innerHTML = jsonFromResponse;
 }
 
-function getRandomQuote(jsonFromResponse) {
-    const randomIndex = Math.floor(Math.random() * jsonFromResponse.quotes.length);
-    const randomQuote = jsonFromResponse.quotes[randomIndex];
-    return randomQuote;
+// function getRandomQuote(jsonFromResponse) {
+//     const randomIndex = Math.floor(Math.random() * jsonFromResponse.quotes.length);
+//     const randomQuote = jsonFromResponse.quotes[randomIndex];
+//     return randomQuote;
+// }
+
+async function showMessages() {
+    const responseFromServer = await fetch('/show-messages');
+    const jsonFromResponse = await responseFromServer.json();
+    console.log(jsonFromResponse);
+
+    const messagesContainer = document.getElementById('messages-container');
+    messagesContainer.innerHTML = jsonFromResponse;
```

Comment:
jsonFromResponse is an array, so you could instead create multiple div/p elements for each message in the array. Right now it'll print all messages in one line: https://xchen-sps-summer22.appspot.com/messages.html